l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -290,17 +290,20 @@ def _setStabileOrganizzazione(self, CedentePrestatore, company):
 
     def _setRea(self, CedentePrestatore, company):
 
-        if company.fatturapa_rea_office and company.fatturapa_rea_number:
+        if (
+            company.fatturapa_rea_office and company.fatturapa_rea_number and
+            company.company.fatturapa_rea_liquidation
+        ):
+            # The required fields for IscrizioneREA (not required) are
```

Comment:
Il `(not required)` dopo `IscrizioneREA` è corretto?

Reply:
Sì, perchè il nodo `IscrizioneREA` non è obbligatorio. Ma, se presente, `Ufficio, NumeroREA and StatoLiquidazione` sono obbligatori